### PR TITLE
Add the worldPosition property to the PersonDetection model

### PR DESCRIPTION
**Pull request by micaelmbagira**
This new property represents the person's coordinates relative to the room's coordinate system as calibrated via tec engine. I've also added these properties to the test utils so that it's possible to create mock data with a given position.

### CODE REVIEW

File: src/poi/test-utils/common.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export interface PersonOptions {
   z?: number;
   u?: number;
   v?: number;
+  worldPosition?: { x: number; y: number; z: number };
```
Comment (fbobbio) [inline src/poi/test-utils/common.ts]: @micaelmbagira do you already have a strategy for the future regarding this vs the normal x,y,z?

Comment (micaelmbagira) [reply src/poi/test-utils/common.ts]: I think we should remove this `worldPosition` and only use `x`, `y`, `z`. What do you think?

Comment (fbobbio) [reply src/poi/test-utils/common.ts]: @micaelmbagira so as a next step we basically take over the `x, y, z` for the new coordinates and impact all the components depending on this, correct?

We probably need to pipe this to the backlog.

Comment (micaelmbagira) [reply src/poi/test-utils/common.ts]: I would go for that yes

**Pull request merged**